packages/template-retail-react-app/app/pages/account/wishlist/partials/wishlist-secondary-button-group.jsx

```diff
@@ -57,6 +61,9 @@ const WishlistSecondaryButtonGroup = ({productListItemId, onClick = noop}) => {
 
     const showRemoveItemConfirmation = () => {
         modalProps.onOpen()
+        // After we remove an item from the wishlist
+        // we need to place focus to the next logical place for accessibility
+        focusElementOnRemove?.current?.focus()
```

Comment:
This goes after onOpen. The user can still click no,and item will not be removed. I think we should only focus on the next logical place only if this item is successfully removed, if it is not removed, the focus should stay the same to avoid confusion for screen reader